.github/pull_request_template.md

```diff
@@ -1,41 +1,36 @@
-## Issues
```

Comment:
This is a really bad diff - probably easier to just read the final file in its full state